SRIOV-config/UNIV/full-install.sh

```
@@ -148,10 +148,24 @@ function add_mc_realloc {
 }
 
-if [[ ${NIC_MODEL} == CX* ]]; then
-   echo "next, add add_mc_realloc"
-   prompt_continue
-   add_mc_realloc
+# Probe whether the NIC can allocate VFs without pci=realloc.
+WORKER_ARR=(${WORKER_LIST})
+NUM_VFS=$(awk '/numVfs:/{print $2}' templates/${NIC_MODEL}/sriov-node-policy.yaml.template)
+export REGULUS_INTERFACE_PCI=$(exec_over_ssh ${WORKER_ARR[0]} "ethtool -i ${REGULUS_INTERFACE}" | awk '/bus-info:/{print $NF;}')
+echo "Probing VF allocation on ${REGULUS_INTERFACE} (${REGULUS_INTERFACE_PCI}) ..."
+probe_err=$(exec_over_ssh ${WORKER_ARR[0]} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)